SECURITY.md

```diff
@@ -1,84 +1,93 @@
+-   [Initial Contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact): how to establish initial contact with Yearn's security team.
+-   [Giving Details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details): what details to include with your vulnerability disclosure after having received a response to your initial contact.
+-   [Setting Dates](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#setting-dates): how to agree on timelines for releasing updates and making details of the issue public.
 
-**Medium** - an issue that may result in a loss of greater than \$10k but less than \$1m of funds locked in Yearn, or cause either extreme user dissatisfaction or damage to the ecosystem.
+Any expected deviations and necessary clarifications around the standard are explained in the following sections.
 
-**Low** - an issue that may result in user dissatisfaction but less than \$10k loss of funds locked in Yearn.
+## Receiving Disclosures
 
-**Note** - an issue that may result in minimal user dissatisfaction and no financial loss, but could be rectified to improve the Yearn ecosystem.
+Yearn is committed to working with researchers who submit security vulnerability notifications to us, to resolve those issues on an appropriate timeline, and to perform a coordinated release, giving credit to the reporter if they would so like.
 
-### Likelihood Classifications
+Please submit issues to **all** of the following main points of contact for
+security related issues according to the
+[initial contact](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#initial-contact)
+and [giving details](https://github.com/RD-Crypto-Spec/Responsible-Disclosure#giving-details)
+guidelines.
 
-**Certain** - exploitable by anyone, with no preconditions required for success.
+For all security related issues, Yearn has 3 main points of contact:
 
-**Likely** - exploitable by anyone, under certain conditions (e.g., user with x amount of funds locked, or exploitable only over a longer period of time).
+| Contact                | Public key                                                                                                   | Email                             | Keybase                                         |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------ | --------------------------------- | ----------------------------------------------- |
+| milkyklim              | [PGP](https://github.com/iearn-finance/yearn-security/blob/master/keys/milkyklim.asc)                        | y at milkyklim.com                | [@milkyklim](https://keybase.io/milkyklim/chat) |
+| Doggie Boy / fubuloubu | [PGP](https://github.com/iearn-finance/yearn-security/blob/master/keys/publickey.yearn-security%40pm.me.asc) | yearn-security at pm.me           | N/A                                             |
```

Comment:
@fubuloubu Doggie, you have to change the path to your PGP in another repo. See https://github.com/iearn-finance/yearn-security/issues/6

Reply:
As discussed, Doggie will add his PGP later.